x-pack/plugins/infra/public/pages/metrics/layouts/container.ts

```diff
@@ -16,35 +17,47 @@ import {
 export const containerLayoutCreator: InfraMetricLayoutCreator = theme => [
   {
     id: 'containerOverview',
-    label: 'Container',
+    label: i18n.translate('xpack.infra.containerMetrics.containerLabel', {
+      defaultMessage: 'Container',
+    }),
     sections: [
       {
         id: InfraMetric.containerOverview,
-        label: 'Overview',
+        label: i18n.translate('xpack.infra.containerMetrics.overviewSectionLabel', {
```

Comment:
This would be the "overview" section in this layout, so how about `xpack.infra.metricDetailPage.containerMetricsLayout.overviewSection.sectionLabel`?